src/code/circuit_python/motor.md

```diff
@@ -0,0 +1,232 @@
+Another property called `actuation_range` tells the `Servo` object how far
+the physical servo is able to move, so the code knows how to convert our
+angles into PWM signals.
+
+For controlling motors, we can use the `ContinousServo` class in
+`adafruit_motor.servo`. It has a property called `throttle` which controls
+the speed of the motor. `throttle` can be set to any value between -1.0
+and 1.0. Positive numbers will make the motor spin one direction and
+negative numbers will make the motor spin the other direction. Zero
+throttle will stop the motor.
+
+Both of these objects are created by giving them a `PWMOut` object
+connected to a pin. Optionally, you can also tell them the minimum and
+maximum pulse widths in milliseconds.
+
+API documentation for the `adafruit_motor` module is available
+[here](https://docs.circuitpython.org/projects/motor/en/latest/api.html).
+
+
+## Looping Over Numbers
+
+A common need in programs is looping over a range of numbers. This could
+be just because we need to repeat some action a certain number of times,
+or to loop through the elements in a list of items, for example. In
+Python, we can do this with a `for` loop and the `range` function.
+
+A `for` loop, in Python, runs the body of its loop once for each element a
+container. If you give it a list of numbers like `[1, 2, 3, 4]`, the loop
+will run four times.
+
+```Python
+# This loop will run 4 times
+for i in [1, 2, 3, 4]:
+  # i holds the current element from the list
+  print(i)
+# This code will output:
+# 1
+# 2
+# 3
+# 4
+```
+
+Usually, we don't want to list out each number in the range one by one.
+Instead, we create a `range` object. A `range` object can be created in
+one of three ways:
+
+```Python
+# Create a range with the number of values it should hold
+# This range will hold the numbers 0 through 9
+range(10)
+
+# Create a range with a start and stop value
+# This range will hold the numbers 5 through 19
+range(5,20)
+
+# Create a range with start, stop, and step values
+# This range will hold even numbers from 0 through 98
+range(0, 100, 2)
+```
+
+Using a `range` in a for loop, we can make a loop that runs 10 times:
+
+```Python
+for i in range(10):
+  print(i)
+```
+
+## Controlling a Motor With Gizmo
+
+Now let's write a program that controls a motor connected to our Gizmo.
+This program will run the motor through at different speeds. To get
+started, connect a motor to your Gizmo using the Vex MC29 motor controller
+on motor port 1.
+
+To start our program, we'll import the modules we need to use.
+
+```Python
+import pwmio
+import time
+import adafruit_motor.servo
+import circuitpython_gizmo
+```
+
+For the setup portion of our code, we just need to setup a `Gizmo` object
+and an object to control the motor. To make sure we're creating the right
+kinds of signals for the Vex MC29 motor controller, we need to set the PWM
+frequency to 50 Hz, the min pulse width to 1000 milliseconds, and the max
+pulse width to 2000 milliseconds.
```

Comment:
These are magic numbers as written.  A sentence needs to be added here to explain that these numbers came from the manufacturer datasheet.  You could put this in a `TIP` block if you prefer.